zerver/tests/test_custom_profile_data.py

```diff
@@ -434,9 +435,16 @@ def test_update_profile_data_successfully(self) -> None:
 
         iago = self.example_user('iago')
         expected_value = {f['id']: f['value'] for f in data}
+        expected_rendered_value = {}  # type: Dict[Union[int, float, str, None], Union[str, None]]
+        for f in data:
+            if CustomProfileField.objects.get(id=f['id']).is_renderable():
```

Comment:
Maybe this sound not necessary enough as these are tests, but I am posting here anyway. 😄  here is the diff of changes I made locally, which will make it easier to read and don't need to fetch field each time.
```
diff --git a/zerver/tests/test_custom_profile_data.py b/zerver/tests/test_custom_profile_data.py
index 79778b56a..a1e581642 100644
--- a/zerver/tests/test_custom_profile_data.py
+++ b/zerver/tests/test_custom_profile_data.py
@@ -426,6 +426,7 @@ class CustomProfileFieldTest(ZulipTestCase):
             data.append({
                 'id': field.id,
                 'value': value,
+                'field': field,
             })
 
         # Update value of field
@@ -437,7 +438,7 @@ class CustomProfileFieldTest(ZulipTestCase):
         expected_value = {f['id']: f['value'] for f in data}
         expected_rendered_value = {}  # type: Dict[Union[int, float, str, None], Union[str, None]]
         for f in data:
-            if CustomProfileField.objects.get(id=f['id']).is_renderable():
+            if f['field'].is_renderable():
                 expected_rendered_value[f['id']] = bugdown_convert(f['value'])
             else:
                 expected_rendered_value[f['id']] = None
```

Reply:
This is a pretty good optimization, thanks for pointing it out!